### PR TITLE
Update changelog

### LINKED ISSUE #970: Update changelog
Before releasing, we will need to update the changelog with all changes, we should also consider making this task part of every PR that either:
- fixes a bug
- adds a new feature
- breaks an existing API

### PR DESCRIPTION
**Pull request by nklincoln**
Closes #970 

Signed-off-by: nkl199@yahoo.co.uk <nkl199@yahoo.co.uk>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,30 @@
+## 0.4.0
+
+* Core changes
+  * [Deprecated] Deprecation of Sawtooth connector
```
Comment (davidkel) [inline CHANGELOG.md:4]: Guess you have more than just deprecated them now :-)

**Review by davidkel — CHANGES_REQUESTED** [review]
Some modifications required as 0.4 not quite there yet.

File: CHANGELOG.md (hunk 1 above)
Comment (nklincoln) [reply CHANGELOG.md:4]: true 😂  - have put out feelers on the RC channels ... will hopefully gather some interest 🤷

Comment (nklincoln) [reply CHANGELOG.md:4]: no replies on the channels :(

**Review by davidkel — APPROVED** [review]

**Pull request merged**